clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs

```diff
@@ -0,0 +1,12 @@
+module Test.Tasty.Hedgehog.Extra (testPropertyXXX) where
```

Comment:
I know we've been historically bad about this, but can you put in a haddock module header with the copyright?